scripts/sample_vcf_to_zarr.py

```diff
@@ -0,0 +1,167 @@
+                        required=True,
+                        help="path to output Zarr directory.")
+    parser.add_argument("--sample",
+                        required=True,
+                        help="Sample identifier.")
+    parser.add_argument("--contig",
+                        required=True,
+                        action='append',
+                        dest='contigs',
+                        help="Contig to extract. Multiple values may be provided.")
+    parser.add_argument("--field",
+                        required=True,
+                        action='append',
+                        dest='fields',
+                        help="Field to extract, e.g., 'variants/MQ' or 'calldata/GT'. Multiple "
+                             "values may be provided.")
+    parser.add_argument("--compress-algo",
+                        help="Blosc compression algorithm.  Choose from [zstd, blosclz, lz4, "
+                             "lz4hc, zlib, snappy].",
+                        default="zstd")
+    parser.add_argument("--compress-level",
+                        help="Compression level. Choose integer from [0, 9].",
+                        default=1,
+                        type=int)
+    parser.add_argument("--compress-shuffle",
+                        help="Type of data shuffling used to obtain contiguous runs of same "
+                             "values for improving compression. Choose integer value from "
+                             "NOSHUFFLE (0), SHUFFLE (1), BITSHUFFLE (2) or AUTOSHUFFLE (-1). "
+                             "If -1 (default), bit-shuffle will be used for buffers with "
+                             "itemsize 1, and byte-shuffle will be used otherwise.",
+                        default=-1,
+                        type=int)
+    parser.add_argument("--alt-number",
+                        help="Expected maximum number of alternate alleles.",
+                        default=3,
+                        type=int)
+    parser.add_argument("--tabix",
+                        help="Path to tabix executable v0.2.5+.",
+                        default="tabix")
+    parser.add_argument("--chunk-length",
+                        help="Chunk length in number of variants.",
+                        default=2**18,
+                        type=int)
+    parser.add_argument("--chunk-width",
+                        help="Chunk width in number of samples.",
+                        default=64,
+                        type=int)
+    parser.add_argument("--log",
+                        help="Path to logfile, stdout or stderr. Default: stdout.",
+                        default="stdout")
+    parser.add_argument("--zip", action="store_true",
+                        help="If flag exists, entire zarr folder is zipped with no "
+                             "compression, and the original zarr folder is deleted. "
+                             "The zip file name is output_zarr_path appended with '.zip'.")
```

Comment:
nitpick.  The actual name (that presented to the user) of the argument is 'output'